src/content/developers/docs/consensus-mechanisms/poa/index.md

```diff
@@ -0,0 +1,69 @@
+
+## Prerequisites {#prerequisites}
+
+To better understand this page, we recommend you first read up on [transactions](/developers/docs/transactions/), [blocks](/developers/docs/blocks/), and [consensus mechanisms](/developers/docs/consensus-mechanisms/).
+
+## What is proof-of-authority (POA)? {#what-is-poa}
+
+Proof-of-Authority is a modified version of [proof-of-stake](/developers/docs/consensus-mechanisms/pos/) that is a reputation-based consensus algorithm instead of stake-based mechanism in POS. The term has been introduced for the first time in 2017 by Gavin Wood, and this consensus algorithm has been mostly used by private chains, testnets and local development networks, as it overcomes the need for high quality resources as POW does, and overcomes the scalability issues with POS by having small subset of nodes storing the blockchian and producing blocks.
+
+The Proof-of-Authority simply requires to trust a subset of authorized signers that are set in the [genesis block](/glossary/#genesis-block), they have the same power in all situations as signers, no signer has exclusive privileges than others in the blockchain as you might think(unless new implementation follow this concept is introduced), the idea behind reputation staking is every authorized signer is well-known to everyone through things like KYC, or by having a well known org shouting they will be the only signer in their blockchain, this way if someone did something wrong, it can be detected who and what their identity is, i.e there will be no attacks on the consensus mechanism by anonymous signers.
+
+The Proof-of-Authority has different implementations, but the ethereum standard implementation is clique that implements [eip-225](https://eips.ethereum.org/EIPS/eip-225), it has the most developer-friendly and easy to implement standard that all clients syncing types are supported, unlike a previously used implementation (Aura) that was implemented in openethereum, which fast,light syncing are not supported because smart contracts are used to store the authorized signers, clique instead only reuse obsoleted headers to solve the issues related to voting to add and remove signers.
+
+## how it works {#how-it-works}
+
+clique standard simply make a use of the obsolete headers: extradata that was used only for fun, miner became obsolete as there are no miners in POA, and nonce became also obsolete because the authorities the order of users transactions, extradata is used to store the authorized signers set, miner and nonce are used to vote for adding or removing signers from the network, where every epoch transition flushes all pending votes. The block proposal is selected sequentially for each block, but in fact any other authorized signer can propose block even if it's not his turn, but they have limit of signing only one block between K blocks when N are the authorized signers.
```

Comment:
Think this needs a bit of a rework... As a user who lands on this page, new to the topic of PoA and in the "How it works" section, I would probably want to see this section introduced in a way that lays out the different important parts about how PoA works, and then explain them. This currently opens with a run-on statement that feels like we're in the middle of an explanation already.

Looks like we're covering:
1. how the block data is utilized (more technical, consider not leading with this one)
1. adding/removing signers from the network
1. proposing blocks

I would consider re-writing and potentially expanding on this a bit, using those three items as key points to cover. With enough info, we could turn these into three sub-headers, but it may just need an introduction statement and some hard-stops to break up the sentences. 

example intro sentence:
"When looking at how proof-of-authority works, there are some differences to other consensus mechanisms as to how blocks are structured, how blocks are proposed, who can propose them, and how to add or remove users from that list."
... Something like that where you could then go into details about these after framing the discussion.